### PR TITLE
Log events

### PR DESCRIPTION
**Pull request by henel677**
Adds generation of log event EVENT_RUNTIME_IS_READY

### CODE REVIEW

File: src/runtime/client-event-manager.ts [hunk 1]
```diff
@@ -113,11 +113,15 @@ export class ClientEventManager implements ClientEventManagerApi {
...
+  logEvent(
+    event: ClientEvent,
+    eventParams?: ClientEventParams,
+    eventTime?: number
+  ) {
     validateEvent(event);
 
     // Always use current timestamp.
-    event.timestamp = Date.now();
+    event.timestamp = eventTime == null ? Date.now() : eventTime;
```
Comment (ChrisAntaki) [inline src/runtime/client-event-manager.ts:124]:
```suggestion
    event.timestamp = eventTime ?? Date.now();
```

File: src/runtime/client-event-manager.ts [hunk 2]
```diff
@@ -113,11 +113,15 @@ export class ClientEventManager implements ClientEventManagerApi {
...
 
-  logEvent(event: ClientEvent, eventParams?: ClientEventParams) {
+  logEvent(
+    event: ClientEvent,
+    eventParams?: ClientEventParams,
+    eventTime?: number
+  ) {
     validateEvent(event);
 
     // Always use current timestamp.
```
Comment (ChrisAntaki) [inline src/runtime/client-event-manager.ts:123]: Should this comment change?

File: src/runtime/client-event-manager.ts (hunk 1 above)
Comment (henel677) [reply src/runtime/client-event-manager.ts]: done

File: src/runtime/client-event-manager.ts (hunk 2 above)
Comment (henel677) [reply src/runtime/client-event-manager.ts]: done

**Pull request merged**